packages/nx/bin/nx.ts

```diff
@@ -31,7 +32,8 @@ function main() {
   require('nx/src/utils/perf-logging');
 
   performance.mark('loading dotenv files:start');
-  loadDotEnvFiles();
```

Comment:
I don't think this has to change? :thinking: This is our own function

Reply:
typo, changed back